frame/composable-traits/src/lib.rs

```diff
@@ -24,6 +24,7 @@ pub mod governance;
 pub mod lending;
 pub mod liquidation;
 pub mod math;
+pub mod mosaic;
```

Comment:
may be mod bridges mod etheriym?

Reply:
if that is not only eth, may be eth compatible?
cause for cosmos and parachains we will have different bridges afaik